### PR TITLE
Bump Cosmos SDK to v0.46.3

### PR DESCRIPTION
**Pull request by faddat**
## Description

This PR contains two others:

* #2452 
* #2568

Changelog coming from car to lunch!

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,13 +2,17 @@ go 1.18
...
 
+// Dragonberry ics23 patch
+replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
 	github.com/confio/ics23/go v0.7.0
-	github.com/cosmos/cosmos-sdk v0.46.2
+	github.com/cosmos/cosmos-sdk v0.46.4-0.20221021120231-7f88c60e70d1
```
Comment (crodriguezvega) [inline go.mod:15]: Shouldn't this be simply `github.com/cosmos/cosmos-sdk v0.46.3`?

**Comment by crodriguezvega** [comment]
Thanks, @faddat. Can you also please add a changelog entry?

**Comment by catShaark** [comment]
@crodriguezvega, this is not ready to be merged yet. I think we have to address [this PR](https://github.com/cosmos/ibc-go/pull/2568) first

File: go.mod (hunk 1 above)
Comment (faddat) [reply go.mod:15]: No, still working through some things.  Think it is best for now to work from the last commit in v0.46.*

**Comment by faddat** [comment]
I can get a change log entry on here in a few hours

File: go.mod (hunk 1 above)
Comment (crodriguezvega) [reply go.mod:15]: But will this eventually import v0.46.3 before we merge?

**Comment by crodriguezvega** [comment]
> @crodriguezvega, this is not ready to be merged yet. I think we have to address [this PR](https://github.com/cosmos/ibc-go/pull/2568) first

Should we make it a draft PR then?

**Comment by crodriguezvega** [comment]
I think we should take out the changes of #2452 from this PR. Those changes relay on an API breaking change of the `SendPacket` API which will be released in v6.0.0. But we would also like to release the bump of the SDK to v0.46.3 in v5.0.x and v5.1.x, so in order to be able to backport this PR to those release branches we cannot bring the API breaking change.

**Comment by faddat** [comment]
ok.

**Comment by faddat** [comment]
Will do v5 manually.

**Pull request closed without merging**